crates/core/tedge_api/src/mqtt_topics.rs

```diff
@@ -728,6 +734,7 @@ pub enum ChannelFilter {
     AlarmMetadata,
     AnyCommandMetadata,
     CommandMetadata(OperationType),
+    Health,
```

Comment:
This comes as a surprise to see that `status/health` was not already listed among all the MQTT channels!